CITATION.cff

```diff
@@ -50,9 +51,50 @@ authors:
     family-names: Koller
   - given-names: Asher
     family-names: Merrill
+  - given-names: Lutz
+    family-names: Rastatter
   - given-names: Ashton
     family-names: Reimer
   - given-names: Albert Y.
     family-names: Shih
   - given-names: Amanda
-    family-names: Stricklan
\ No newline at end of file
+    family-names: Stricklan
+references:
+, Austin, TX, June 28 - July 3 2010. 
+  - type: conference-paper
+    scope: Cite this paper for as a general reference for the package and its history.
```

Comment:
```suggestion
    scope: Cite this paper as a general reference for the package and its history.
```

Reply:
I'm fine with that ask in the `CITATION.cff`. People should be citing code and paper anyway, so leaving the original paper in doesn't ask much more. It also means there's an old citation to show longevity (because years often show up in text where titles won't). I can just keep the new paper in the actual docs though...

Reply:
Eh, leave it as is, I wanted to bring it up but I agree on the longevity.